Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchableUnboundedPlane.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// in order to get PointerDown and PointerUp events whenever a PokePointer touches the
     /// attached collider through this surface.
     /// </summary>
-    public class NearInteractionTouchableUnboundedPlane : BaseNearInteractionTouchable
+    public class NearInteractionTouchableUnboundedPlane : ColliderNearInteractionTouchable
```

Comment:
I'm not sure this class makes sense any more. It's original intention was to allow for a plane of infinite size that would be touchable. However, it seems like the poke pointer will always dispatch touchUp events once a pointer has passed some distance threshold. Additionally, I don't think the current behavior actually allows a touchable to be infinitely large. Finally, it makes the definition of "NearInteractionTouchable" a bit more muddled since NearInteractionTouchable is really a touchable surface of bounded size, which is essentially the same thing that this class is. 

How do you feel about removing it? I don't think anything except the touchable sample uses it.

Reply:
It also was not released in RC2 so is not a breaking change.

Reply:
I too am happy with the removal of this class. I was unsure how it should be distinct when adding it. As long as touchable are tied to colliders, functionally this is going to behave like `NearInteractionTouchable`. 
Currently, the only distinguishing features of this class is the distance function and cursor behaviour, but neither align with expectations as the collider is not infinite.

Reply:
I have removed this class now.